.github/workflows/sync-mirror.yml

```
@@ -0,0 +1,23 @@
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  mirror_repo:
+    if: github.repository == 'djmaxus/matlab-repo-init.git'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: webfactory/ssh-agent@v0.9.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
```

I would set a more specific name for the secret as long as it has a limited purpose, i.e., keeping the fork for `matlab-repo-init` in sync.

```suggestion
          ssh-private-key: ${{ secrets.MATLAB_TEMPLATE_MIRROR_SSH_PRIVATE_KEY }}
```